connector-sdk/core/src/main/java/io/camunda/connector/api/inbound/webhook/VerifiableWebhook.java

```diff
@@ -32,12 +33,17 @@ public WebhookHttpVerificationResult(Object body) {
       this(body, null, 200);
     }
 
-    public WebhookHttpVerificationResult(Object body, int statusCode) {
+    public WebhookHttpVerificationResult(Object body, Integer statusCode) {
```

Comment:
Why not create a helper method like 
```
static WebhookHttpVerificationResult.ok(Object body) {
   ...
}```

Reply:
A matter of style

Reply:
You would not need the extra code in the `statusCode`

Reply:
> You would not need the extra code in the `statusCode`

We still would need that tho. if the verification expression does not  specify `statusCode: xxx`, it will still evaluate to `null`. Constructors are only used as quality of life improvement mostly for tests, and in this case I think it's a matter of taste.

The only option to remove lines in the `statusCode()` is to add `if-else` checks upstream somewhere, but I would like to avoid that and encapsulate it here.

Reply:
I think we are mixing the custom implementation with the expression here.
When evaluating the expression you could analyze the data and create a record accordingly no?

Reply:
It's evaluated automatically. If the expression is `= {body:{"key":"value"}}`, then it will create an object with `null` as status code and headers. This is beyond of my control. Custom constructor is an option but also a custom getter. Unfortunatelly, we cannot use kotlin style default init like `record Data(body, statusCode=200)`.

Reply:
@igpetrov You could map to to a `Map<K,V>` and evaluate and construct the result manually no?